source/extensions/omni.isaac.lab/omni/isaac/lab/envs/manager_based_env_cfg.py

```diff
@@ -95,3 +96,10 @@ class ManagerBasedEnvCfg:
 
     Please refer to the :class:`omni.isaac.lab.managers.EventManager` class for more details.
     """
+
+    live_visualizer: object = None 
```

Comment:
for type hinting, can that one here has the class `DefaultManagerBasedEnvLiveVisCfg` that you imported in the script ?